yarn.lock

```diff
@@ -2096,11 +2096,12 @@
     js-cookie "^2.2.1"
     whatwg-fetch "^3.0.0"
 
-"@guardian/discussion-rendering@^0.3.1":
-  version "0.3.1"
-  resolved "https://registry.yarnpkg.com/@guardian/discussion-rendering/-/discussion-rendering-0.3.1.tgz#265eff7d72b6e06d728cdfc7d5937b4e0bb314bf"
-  integrity sha512-y7ZWVqGvRKKjL7AwCGqSBI0f+Ak0Wc5PBLJw/iFTZqPJwX6Xgv+R/KWkFu1B/0hZ3hfFEAKIxF8GHd4CEI04LQ==
+"@guardian/discussion-rendering@^0.6.0":
+  version "0.6.0"
+  resolved "https://registry.yarnpkg.com/@guardian/discussion-rendering/-/discussion-rendering-0.6.0.tgz#09b7254161d7f31fbb150cf9353bb6524062ad33"
+  integrity sha512-Rd6JFORlfqVvXMCoACKrMKUW1pRQeKnhoEMBuwocnraofNhhBLllUdC0JkrsloRSp7aTsWpj8VkGCj1JdMb9iw==
   dependencies:
+    "@guardian/src-link" "^0.15.1"
     regenerator-runtime "^0.13.3"
```

Comment:
I’m not sure, should this be a peerDependency in discussion rendering?

Reply:
Done https://github.com/guardian/discussion-rendering/pull/164